### ISSUE TITLE
  Is Vue currently supported?

### ISSUE THREAD

**Issue by liuzheng2000** [crabbly/Print.js#623]
Is Vue currently supported?
How do I use it in Vue and print JSON In Vue?

**Comment by borisdmt** [comment]
yes its working with vue3. I am currently working with quasar and this

**Comment by bobolat** [comment]
> yes its working with vue3. I am currently working with quasar and this

does the css working also? i tried mine quasar and vue 3 also..but some css not working like flex and  font size doesnt effect sigh

**Comment by rafaeusilvar22** [comment]
@bobolat Did you find any solution? As I am experiencing the same problem

**Comment by bobolat** [comment]
> @bobolat Did you find any solution? As I am experiencing the same problem

yes i have..turn out if i just hide the div or whatever box you want to print, its working normal lol, about font size,,at setting for print just put font_size = '' will do

**Comment by rafaeusilvar22** [comment]
Got it. I'm trying to print a page that contains components with different quasar classes, which end up being ignored when generating the pdf. Generating a great deal of disorganization. :(
